examples/tv-casting-app/tv-casting-common/commands/clusters/ModelCommand.cpp

```diff
@@ -82,6 +82,11 @@ void ModelCommand::Shutdown()
     mOnDeviceConnectionFailureCallback.Cancel();
 }
 
+void ModelCommand::ClearICDEntry(const chip::ScopedNodeId & nodeId)
```

Comment:
This file has `using namespace chip`.  Why the prefix?

Reply:
https://github.com/project-chip/connectedhomeip/pull/33690
follow-up here